src/oss-characteristics/CharacteristicTool.cs

```diff
@@ -267,7 +267,7 @@ public async Task RunAsync(Options options)
                                 {
                                     { target, analysisResult }
                                 };
-                                var encodedName = Uri.EscapeUriString(Path.GetDirectoryName(target) ?? "unknown");
+                                var encodedName = Uri.EscapeUriString(target ?? "unknown");
```

Comment:
@scovetta Why is this being escaped here?

Reply:
This results in the output being all % symbolly.

Reply:
I don't think the encoding is needed here.

Reply:
Thanks. I’ll remove.